.github/workflows/build-http-broker-containers.yaml

```diff
@@ -0,0 +1,73 @@
+name: "Build HTTP Protocol Brokers"
+
+on:
+  push:
+    branches:
+      - http-extensibility
+
+jobs:
```

Comment:
consider using strategy-matrix here to reduce steps replication

Reply:
See previous comment and https://github.com/deislabs/akri/commit/f8d9de76067cf547cdf10a20c89c060f996f9b0e

Reply:
can the same matrix solution be applied here?

    strategy:
      matrix:
        include:
          - component: device
            dockerfile: device
          - component: discovery
            dockerfile: discovery
          - component: grpc-broker
            dockerfile: grpc.broker
          - component: grpc-client
            dockerfile: grpc.client